cypress/support/commands.ts

```diff
@@ -189,6 +189,7 @@ Cypress.Commands.add("seedPolicies", (team = "") => {
         "Checks to make sure that the Filevault feature is enabled on macOS devices.",
       resolution:
         "Choose Apple menu > System Preferences, then click Security & Privacy. Click the FileVault tab. Click the Lock icon, then enter an administrator name and password. Click Turn On FileVault.",
+      platform: "darwin,linux",
```

Comment:
@lukeheath Yes, because according to our working definition of compatible, the checker just looks whether the tables exist in the osquery schema and the `WHERE` clause doesn't get factored into the result. The checker was really designed to work on user-generated queries where our caveat about the precision of the checker (in the tooltip) made sense. The UX disconnect comes up now because we have introduced these template policies that narrow the query based on platform-specific columns like `filevault_status`.  If we are going to keep adding example policies that narrow based on columns, we definitely need to spec a ticket to figure out how/where we are going to include this level of detailed platform compatibility info for columns. AFAIK that level of detail doesn't exist currently in the osquery schema. So perhaps we may want to have our devrel team do some work to contribute that to the osquery project.

Reply:
@noahtalerman Heads up about potential confusion with this feature and template policies ^

Reply:
> AFAIK that level of detail doesn't exist currently in the osquery schema

Really good point. I think you're correct about the above. 

> narrow the query based on platform-specific columns

I think this limitation in the compatibility checker is ok for now. However, we'll likely want to come back to address this when we have better docs that call out tables with columns that differ in compatibility.

I called this out in the following "Port osquery schemas" issue in a comment here: https://github.com/fleetdm/fleet/issues/4472#issuecomment-1074227272